File: app/src/main/java/io/github/pshegger/infobook/model/FileData.java
package io.github.pshegger.infobook.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by pshegger on 2015.10.04..
 */
public class FileData {
    @SerializedName("id")
    String id;
    @SerializedName("icon")
    String url;

    public String getUrl() {
        return url;
    }
}

File: app/src/main/kotlin/io/github/pshegger/infobook/model/FileData.kt
package io.github.pshegger.infobook.model

import com.google.gson.annotations.SerializedName

/**
 * Created by pshegger on 2015. 11. 10..
 */
class FileData(
        @SerializedName("id") val id: String,
        @SerializedName("icon") val url: String)